block_function.rb

```diff
@@ -0,0 +1,13 @@
+
+
+def do_calc(a, b)
+    yield(a, b)
```

Comment:
```suggestion
    yield(7, 9)
```

The assignment actually wanted this instead. Can you put in this change and clean up the rest of the file? 7 and 9 shouldn't appear after this line.